netket/graph/lattice.py

```diff
@@ -538,6 +490,37 @@ def position_from_basis_coords(self, basis_coords: CoordT) -> PositionT:
             raise KeyError(f"No site found at at least one of {basis_coords}")
         return self.positions[ids]
 
+    def to_reciprocal_lattice(self, ks: Array, strict: bool = False) -> Array:
+        """
+        Returns input wave vectors in units of reciprocal basis vectors **of the
+        simulation box.**
+        Also checks whether the coordinates be integers (if the axis has periodic
+        boundary conditions) or zero (if not):
+        * if `strict` is `True`, throws a `ValueError`
+        * if `strict` is `False`, returns None for invalid coordinates.
+        """
```

Comment:
It is not clear from the docstring to me what exactly this function maps to what. So the input are reciprocal-space vectors (in coordinates dual to the _position space basis_ (not the _primitive vector basis_) and the output is the same reciprocal location in units of the reciprocal basis vectors (that is, the basis dual to the primitive vectors)? Is that correct?